src/allencell_ml_segmenter/scripts/bumpver_handler.py

```diff
@@ -20,36 +20,33 @@ def main() -> None:
     version_components: list[str] = version.split(".")
 
     update_output: subprocess.CompletedProcess
-    # 4 components means we currently have a dev or post version
-    if len(version_components) == 4 and version_components[-1].startswith(
-        "dev"
-    ):
-        if component == "dev":
-            # increment the dev tag (e.g. 1.0.0.dev0 -> 1.0.0.dev1)
+    if "rc" in version_components[-1]:
```

Comment:
Why don't we need to check the # of components now?

Reply:
Seems like that should happen right after we split the string?

Reply:
The standard for RC seems to be `X.X.XrcY` (note the lack of a [period between patch and RC). Bumpver, for some reason, makes you choose a pattern for all tags to match, so in order to get `rc` in the right place, we have to make both rc and post tags follow the no-period pattern. This means that the number of components derived by splitting on periods will always be three now. 

I could also do a fancier regex to split on `(\.|[a-z])+`, but then I'd have to go back to look at the original string anyways to figure out whether its rc or post.